django_project/geocontext/models/context_service_registry.py

```diff
@@ -160,6 +164,12 @@ class ContextServiceRegistry(models.Model):
         max_length=1000,
     )
 
+    is_graph = models.BooleanField(
```

Comment:
You should put this field in the context_group model instead, we will be able to draw a graph from the groups of context.